android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -481,8 +481,8 @@ private void doInitializePlayer(boolean force) {
                     getContext(),
                     adType,
                     MAX_LOAD_BUFFER_MS,
-                    exoDorisPlayerView.getFastForwardIncrementMs(),
-                    exoDorisPlayerView.getRewindIncrementMs(),
+                    (src.getDvrSeekForwardInterval() != 0L) ? src.getDvrSeekForwardInterval() : exoDorisPlayerView.getFastForwardIncrementMs(),
```

Comment:
perhaps we can move them into variables and we can avoid executing `getDvrSeekForwardInterval()` twice

Reply:
Agreed, changed